xoops_trust_path/modules/xelfinder/sql/mysql.sql

```diff
@@ -31,7 +31,7 @@ CREATE TABLE `link` (
   `title` varchar(255) NOT NULL DEFAULT '',
   PRIMARY KEY (`id`),
   UNIQUE KEY `mid_key_val` (`mid`,`param`,`val`),
-  KEY `uri` (`uri`(255)),
+  KEY `uri` (`uri`(200)),
```

Comment:
When creating a key, it is created with the key length changed on the system side, so this change is not necessary.